duckduckgo/macos-browser
# Allow editing duplicate logins

Task/Issue URL: https://app.asana.com/0/1204099484721401/1205087530378369/f

**Description**:
Allow editing duplicates.  (While creating duplicates from the UI is not permited, we may get duplicates via Sync, and need to provide a way for users to edit them)

**Steps to test this PR**:
1.  Add a new login with the following data:

- **User**: daniel
- **Domain**: daniel.com

2. Add another login with the following data

- **User**: daniel2
- **Domain**: daniel.com

3.  Edit the last login, and change the username to "daniel"
4. Save
5. Observe it can be saved
 
6. Try to add a new login via the ui with the following data

- **User**: daniel
- **Domain**: daniel.com

7. Observe a duplicate alert message shows up.



**Note.** Anything with the same username/domain is considered a duplicate.



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
